suite/runsuite.cmake

```diff
@@ -421,7 +422,8 @@ if (NOT cross_riscvxx_linux_only AND NOT cross_aarchxx_linux_only AND NOT cross_
         ")
     endif (DO_ALL_BUILDS)
   endif (ARCH_IS_X86 AND NOT APPLE)
-endif (NOT cross_aarchxx_linux_only AND NOT cross_android_only AND NOT a64_on_x86_only)
+endif (NOT cross_riscv64_linux_only NOT cross_aarchxx_linux_only AND
```

Comment:
This is missing an `AND`.

s/NOT cross_riscv64_linux_only NOT/NOT cross_riscv64_linux_only AND NOT/

Reply:
Done. I will push it later.